docs/modules/MongoDb.md

```diff
@@ -30,6 +31,8 @@ Check out the driver if you get problems loading dumps and cleaning databases.
 * dsn *required* - MongoDb DSN with the db name specified at the end of the host after slash
 * user *required* - user to access database
 * password *required* - password
+* dump_type *required* - type of dump.
+  One of MongoDb::DUMP_TYPE_JS ('js'), MongoDb::DUMP_TYPE_MONGODUMP ('mongodump') or MongoDb::DUMP_TYPE_MONGODUMP_TAR_GZ ('tar-gz'). Default value is MongoDb::DUMP_TYPE_JS.
```

Comment:
Constants can't be used in configuration, so it is better just to list the string values here, the last value should probably be a 'mongodump-tar-gz'